clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -79,6 +81,8 @@ class CheckedRegionFinder : public RecursiveASTVisitor<CheckedRegionFinder>
     bool isUncheckedPtrAcc(QualType Qt, std::set<std::string> &Seen);
     bool isUncheckedStruct(QualType Qt, std::set<std::string> &Seen);
     bool isFunctionBody(CompoundStmt *S);
+    bool isWild(std::set<ConstraintVariable*>&);
+    bool isWild(std::set<FVConstraint*>*);
```

Comment:
Why is one taking a reference and the other taking a pointer?

Reply:
Difference in the interface defined in `ProgramInfo`. `getVariable` returns a set, and `getFuncConstraints` returns a pointer